PR: https://github.com/odoo/odoo/pull/

From: 9a7fd72197799c0815f0d2e0ed4f895f674a053d
From: Adrian Torres
Date: 2017-12-12 14:56:59

Breaking data model changes scores: del:6 + add:0, change matches:
-    social_twitter = fields.Char('Twitter Account')
-    social_facebook = fields.Char('Facebook Account')
-    social_github = fields.Char('GitHub Account')
-    social_linkedin = fields.Char('LinkedIn Account')
-    social_youtube = fields.Char('Youtube Account')
-    social_googleplus = fields.Char('Google+ Account')

Total Changes: 8

[ADD] social_media: new module for social media

This technical dependency module adds settings to res_company for the
company's social media pages.

It is introduced to fix duplicated fields for res_company in website and
mass_mailing, these two modules now require social_media as a dependency

================================= pseudo patch: =================================

--- a/addons/website/models/res_company.py
+++ b/addons/website/models/res_company.py
@@ -7,14 +7,6 @@ from odoo import api, fields, models
 class Company(models.Model):
     _inherit = "res.company"
 
-    # NB: Also defined in mass_mailing
-    social_twitter = fields.Char('Twitter Account')
-    social_facebook = fields.Char('Facebook Account')
-    social_github = fields.Char('GitHub Account')
-    social_linkedin = fields.Char('LinkedIn Account')
-    social_youtube = fields.Char('Youtube Account')
-    social_googleplus = fields.Char('Google+ Account')
-
     @api.multi
     def google_map_img(self, zoom=8, width=298, height=298):
         partner = self.sudo().partner_id
